v1/transform.go

```diff
@@ -0,0 +1,344 @@
+// Copyright (C) 2021 IOTech Ltd
+
+package v1
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/common"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/models"
+	v1Models "github.com/edgexfoundry/go-mod-core-contracts/v2/v1/models"
+)
+
+// TransformToV1DeviceProfile transform v2 profile to v1
+func TransformToV1DeviceProfile(profile models.DeviceProfile) v1Models.DeviceProfile {
+	dto := v1Models.DeviceProfile{
```

Comment:
the variable name should be v1dp, not dto

Reply:
Modified.